index.bs

```diff
@@ -703,6 +703,17 @@ agent MUST run the following steps:
 To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
 <var>transport</var>, a [=URL record=] |url|, and a boolean |dedicated|, run these steps.
 
+1. Let |client| be |transport|'s [=relevant settings object=].
+1. Let |request| be a new [=request=] whose [=request/URL=] is |url|, [=request/client=] is
+   |client|, [=request/policy container=] is a clone of |client|'s
```

Comment:
I don't think you need a clone. You can just write `[=request/policy container=] is |client|'s [=environment settings object/policy container=]`.

For context: we need a clone at other points in order to make all CSP checks with a consistent snapshot of the policies. Since things are synchronous here, it should not be needed.